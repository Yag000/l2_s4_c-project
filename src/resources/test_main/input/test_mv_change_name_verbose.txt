mkdir a
print
mv a b
print
touch c
print
mv c d
print
mkdir b/e
mkdir b/e/f
print
mv b/e/f b/e/g
touch b/e/f
print
mv b/e/f b/e/t
print
