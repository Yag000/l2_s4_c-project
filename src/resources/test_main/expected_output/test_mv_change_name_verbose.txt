Parsing file src/resources/test_main/input/test_mv_change_name_verbose.txt ...
/$ mkdir a
/$ print
/ (D)
\-- a (D)
/$ mv a b
/$ print
/ (D)
\-- b (D)
/$ touch c
/$ print
/ (D)
|-- b (D)
\-- c (F)
/$ mv c d
/$ print
/ (D)
|-- b (D)
\-- d (F)
/$ mkdir b/e
/$ mkdir b/e/f
/$ print
/ (D)
|-- b (D)
|   \-- e (D)
|       \-- f (D)
\-- d (F)
/$ mv b/e/f b/e/g
/$ touch b/e/f
/$ print
/ (D)
|-- b (D)
|   \-- e (D)
|       |-- g (D)
|       \-- f (F)
\-- d (F)
/$ mv b/e/f b/e/t
/$ print
/ (D)
|-- b (D)
|   \-- e (D)
|       |-- g (D)
|       \-- t (F)
\-- d (F)
